pkg/registries/registry.go

```diff
@@ -181,3 +195,63 @@ func createFilter(config Config, log *logging.Logger) Filter {
 
 	return filter
 }
+
+func validateSpecs(log *logging.Logger, inSpecs []*apb.Spec) []*apb.Spec {
+	var wg sync.WaitGroup
+	wg.Add(len(inSpecs))
```

Comment:
@shawn-hurley coming back to this now that the deprovision fire is (hopefully) out. I think we're talking about how the broker should validate Specs that come in, which IMO is outside the scope of this PR. I would like to lodge a card to consider spec validation explicitly, because it's something we need to address. The broker needs to defend against adapters sending bunk Specs.